projects/angular/src/layout/_login.clarity.scss

```diff
@@ -238,6 +238,7 @@
           margin-top: variables.$clr_baselineRem_0_5;
           font-size: variables.$clr_baselineRem_0_583;
           text-align: center;
+          place-content: center;
```

Comment:
As discussed offline, this probably wouldn't be needed when we revert the change for the anchor.

Reply:
corrected in commit bf22d26